src/main/java/com/passbook/sparkeighteen/peristence/entity/ProfileEntity.java

```diff
@@ -33,8 +33,8 @@ public class ProfileEntity {
     @Column(name = "id")
     private Integer id;
 
-    @OneToOne(cascade = CascadeType.ALL)
-    @JoinColumn(name = "user_id", referencedColumnName = "id")
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "profile")
```

Comment:
Is it right?

Reply:
yes i joind the profileEntity with profile object variable of userEntity.

Reply:
@kunalcodess Have a look at it please?

Reply:
updated that change